deku-c/deku-cli/index.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
```

Comment:
Looks like this file escaped .gitignore mistakenly.

Reply:
Yup, done.